x-pack/packages/ml/aiops_components/src/progress_controls/progress_controls.tsx

```diff
@@ -56,11 +65,34 @@ export function ProgressControls({
       </EuiFlexItem>
       <EuiFlexItem grow={false}>
         {!isRunning && (
-          <EuiButton size="s" onClick={onRefresh}>
-            <FormattedMessage
-              id="xpack.aiops.rerunAnalysisButtonTitle"
-              defaultMessage="Rerun analysis"
-            />
+          <EuiButton
+            size="s"
+            onClick={onRefresh}
+            color={shouldRerunAnalysis ? 'warning' : 'primary'}
+          >
+            <EuiFlexGroup>
+              <EuiFlexItem>
+                <FormattedMessage
+                  id="xpack.aiops.rerunAnalysisButtonTitle"
+                  defaultMessage="Rerun analysis"
+                />
+              </EuiFlexItem>
+              {shouldRerunAnalysis && (
+                <>
+                  <EuiFlexItem>
+                    <EuiIconTip
+                      aria-label="Warning"
+                      type="alert"
+                      color="warning"
+                      content={i18n.translate('xpack.aiops.rerunAnalysisTooltipContent', {
+                        defaultMessage:
+                          'Analysis data may be out of date due to selection update. Please rerun analysis.',
```

Comment:
according to the [writing guidelines](https://elastic.github.io/eui/#/guidelines/writing/guidelines) we should avoid using word "please" , cc @szabosteve